raving-remoras/bangazon-site
# Rd recommend product

# Description
From the product detail page, user should be able to recommend a product. If authenticated, a user has a My Recommendations option in navbar, with notifications showing how many products have been recommended. The user can see all recommendations in that view, and delete them (hard delete)

## Number of Fixes
Fixed weird delete button on my products

## Related Ticket(s)
#21 

## Steps to Test Solution

1. python manage.py makemigrations
2. python manage.py migrate
3. Go to a detail page and recommend a product to a user. You should only be able to do so if you get the username correct, otherwise you should get an error message. You can either reference/manipulate the database to see success or recommend to yourself.
4. Give yourself a recommendation and make sure the notification shows up in the nav bar, and that the recommended product shows up in that view


## Testing


## Documentation
